src/common/types/store-data/user-configuration-store.ts

```diff
@@ -11,6 +11,8 @@ export interface UserConfigurationStoreData {
 
     bugService: string;
     bugServicePropertiesMap: IssueFilingServicePropertiesMap;
+
+    adbLocation: string;
```

Comment:
I would do `string | null` with a comment about what `null` implies; it's not technically necessary since we don't have strict null checks enabled, but I think it makes it more clear to the reader.